src/XMakeCommandLine/Microsoft.Build.CommonTypes.xsd

```diff
@@ -3970,81 +4003,6 @@ elementFormDefault="qualified">
             <xs:documentation><!-- _locID_text="AppxValidateStoreManifest" _locComment="" -->Flag indicating whether to validate store manifest.</xs:documentation>
         </xs:annotation>
     </xs:element>
-    
-    <!-- Following properties are declared here to get rid of schema validation warnings,    -->
-    <!-- but are not intended for overriding, so they do not have documentation annotations. --> 
```

Comment:
Correct, as I understand it these are used in Task and Targets and shouldn't really go into project files but were added so that they wouldn't squiggle. Now that we modified the xml language service to opt out of schema validation we don't need them and we don't want them to show up in IntelliSense.